src/ui/setup/dynamic/devtools.ts

```diff
@@ -9,7 +9,9 @@ import * as dbgClient from "devtools/client/debugger/src/client";
 import debuggerReducers from "devtools/client/debugger/src/reducers";
 import * as inspectorReducers from "devtools/client/inspector/reducers";
 // eslint-disable-next-line no-restricted-imports
-import { addEventListener, initSocket, client as protocolClient } from "protocol/socket";
+import { addEventListener, initSocket } from "protocol/socket";
+// eslint-disable-next-line no-restricted-imports
+import { listenForSessionDestroyed, client as protocolClient } from "protocol/socket";
```

Comment:
This is goofy, but all 4 imports being in the same statement causes Prettier to re-format the import to be multi-line, which breaks the Lint rule (bc there's no way to ignore it at that point)